llama-cli/src/main.rs

```diff
@@ -73,8 +72,8 @@ fn infer(args: &cli_args::Infer) {
 
 fn dump_tokens(args: &cli_args::DumpTokens) {
     let prompt = load_prompt_file_with_prompt(&args.prompt_file, args.prompt.as_deref());
-    let (_, vocabulary) = args.model_load.load();
-    let toks = match vocabulary.tokenize(&prompt, false) {
+    let model = args.model_load.load();
```

Comment:
I agree with the change. I had considered doing this a few times before. Since both the model and the vocabulary are meant to be immutable after creation, bundling them into the same struct will hardly cause issues.